app_dart/lib/src/model/ci_yaml/target.dart

```diff
@@ -33,6 +35,18 @@ class Target {
   /// Target prefixes that indicate it will run on an ios device.
   static const List<String> iosPlatforms = <String>['mac_ios', 'mac_ios32'];
 
+  /// Gets dimensions for this [pb.Target].
```

Comment:
Can you expand on what dimensions are? Possibly a link to the swarming documentation